test/spec/formmodel.spec.js

```diff
@@ -419,11 +419,16 @@ describe( 'DeprecatedID value getter', () => {
         model.init();
         expect( model.deprecatedID ).to.equal( '' );
     } );
-    it( 'returns value of deprecatedID node', () => {
+    it( 'returns value of deprecatedID node if the meta block does not have the "http://openrosa.org/xforms" namespace', () => {
         const model = new Model( '<model><instance><data><meta><instanceID/><deprecatedID>a</deprecatedID></meta></data></instance></model>' );
         model.init();
         expect( model.deprecatedID ).to.equal( 'a' );
     } );
+    it( 'returns value of deprecatedID node if the meta block has the "http://openrosa.org/xforms" namespace', () => {
```

Comment:
Not directly testing the issue but it won't hurt to have this (related) test, I think.